src/@batch-flask/electron/shell.service.ts

```diff
@@ -13,10 +13,9 @@ export class ElectronShell {
     }
     /**
      * Show the given file in a file manager. If possible, select the file.
-     * @returns Whether the item was successfully shown.
      */
-    public showItemInFolder(fullPath: string): boolean {
-        return this._shell.showItemInFolder(fullPath);
+    public showItemInFolder(fullPath: string): void {
+        this._shell.showItemInFolder(fullPath);
```

Comment:
Does this no longer return false if unsuccessful?

Reply:
Yup